packages/terra-grid/CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+Terra Grid Changelog
+====================
+
+3.1.0 - 2017-06-20
+------------------
+### Enhancements
```

Comment:
Yeah I like the idea of these sections:
* Bug Fix
* Breaking Changes
* Removals
* Enhancements

Sometimes the only change between a version are dependency updates. I feel like its good to note that in the changelog when we do that so consumers don't need to diff the code between packages to see what changed.
I'd propose calling this section "Dependency Updates"